queries/leo/highlights.scm

```diff
@@ -110,6 +113,18 @@
 
 (comment) @comment @spell
 
+((comment) @comment.todo
+  (#match? @comment.todo "TODO"))
```

Comment:
```suggestion
  (#eq? @comment.todo "TODO"))
```

Do NOT use `#match?`; that does a full (vim) regex match and is much too slow.

Reply:
And is this really helpful? That is usually handled by the `comment` parser, for which you have injections. (No point in every language having these...)

Reply:
Funfact: I did have injections just forgot abut it. So will remove these.

Reply:
rmoved in [bee87a2e12](https://github.com/nvim-treesitter/nvim-treesitter/pull/7506/commits/bee87a2e12e0341c75d14c3197f59060f96f937a)